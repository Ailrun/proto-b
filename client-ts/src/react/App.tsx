import { hot } from 'react-hot-loader';

import { App as AppCore } from './App.core';

const App = hot(module)(AppCore);

export {
  App,
};
