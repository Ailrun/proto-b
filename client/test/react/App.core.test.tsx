import React from 'react';
import { render } from 'react-testing-library';

import { App } from '~src/react/App.core';

describe('App.core', () => {
  it('should be renderable', () => {
    const rendered = render(<App />);

    expect(rendered.container).toMatchSnapshot();
  });
});
