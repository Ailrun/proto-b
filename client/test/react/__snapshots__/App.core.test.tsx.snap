// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.core should be renderable 1`] = `
<div>
  <div>
    Hello World!
  </div>
</div>
`;
